css/style.css

```diff
@@ -1,4 +1,1028 @@
+    align-items: center;
+  }
+
+  .data__figures {
+    font-size: 50px;
+  }
+
+  .data__specifications {
+    display: block;
+  }
+
+  .testdrive__img {
+    display: none;
+  }
+
+  .testdrive__text {
+    position: relative;
+    font-size: 16px;
+    color: #e51937;
+  }
+
+  .testdrive__text::before {
+    content: '';
+    background-image: url("/img/png/steering-wheel-red.png");
+    display: block;
+    position: absolute;
+    width: 32px;
+    height: 32px;
+    left: -45px;
+    top: -8px;
+  }
+
+  .order-btn {
+    width: 221px;
+    height: 55px;
+    background-color: #e51937;
+    color: #fff;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 899px) {
+  .header {
+    padding: 25px 24px 96px 0;
+  }
+
+  .header__logo-img {
+    padding-left: 15px;
+    width: 167px;
+    height: 50px;
+  }
+
+  .menubar__link {
+    display: none;
+  }
+
+  .button {
+    width: 33px;
+    height: 20px;
+  }
+
+  .second {
+    top: 9px;
+  }
+
+  .third {
+    top: 18px;
+  }
+
+  .main-tesla {
+    display: flex;
+    justify-content: center;
+  }
+
+  .tesla-car {
+    max-width: 701px;
+    height: auto;
+    padding: 84px 3px 30px 0;
+  }
+
+  .content__data-car {
+    flex-direction: column;
+    padding: 10px 0 55px 54px;
+  }
+
+  .content__facts {
+    padding: 0 0 54px 0;
+  }
+
+  .data__speed-block {
+    padding-right: 92px;
+  }
+
+  .data__speed-icon {
+    width: 78px;
+    height: 62px;
+    padding: 7px 14px 0 0;
+  }
+
+  .content__road-icon {
+    width: 48px;
+    height: 77px;
+  }
+
+  .data__dimention {
+    align-items: center;
+  }
+
+  .data__figures {
+    font-size: 51px;
+    padding-right: 6px;
+  }
+
+  .data__specifications {
+    display: block;
+  }
+
+  .testdrive__img {
+    display: none;
+  }
+
+  .testdrive__text {
+    position: relative;
+    font-size: 16px;
+    color: #e51937;
+  }
+
+  .testdrive__text::before {
+    content: '';
+    background-image: url("/img/png/steering-wheel-red.png");
+    display: block;
+    position: absolute;
+    width: 32px;
+    height: 32px;
+    left: -45px;
+    top: -8px;
+  }
+
+  .order-btn {
+    width: 221px;
+    height: 55px;
+    background-color: #e51937;
+    color: #fff;
+    font-size: 16px;
+  }
+
+  .arrow {
+    padding-left: 2px;
+  }
+}
+
+@media (min-width: 812px) and (max-width: 899px) {
+  .tesla-bgd {
+    width: 100%;
+    top: -35px;
+    left: 0;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 811px) {
+  .tesla-bgd {
+    top: -22px;
+    max-width: 831px;
+    left: -12px;
+  }
+}
+
+@media (min-width: 700px) and (max-width: 767px) {
+  .tesla-bgd {
+    top: -22px;
+    width: 100%;
+    height: auto;
+  }
+
+  .tesla-car {
+    max-width: 700px;
+    height: auto;
+    padding: 80px 0 30px 0;
+  }
+
+  .content__data-car {
+    flex-direction: column;
+    padding: 11px 0 55px 80px;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 699px) {
+  .tesla-bgd {
+    top: -22px;
+    width: 100%;
+    height: auto;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)